src/tests/ftest/deployment/disk_failure.py

```diff
@@ -147,8 +147,18 @@ def test_disk_fault_to_normal(self):
                 get_dmg_response(
                     self, self.dmg_command.storage_set_faulty, uuid=device["uuid"])
                 # Replace the device with same uuid.
-                get_dmg_response(
-                    self, self.dmg_command.storage_replace_nvme,
-                    old_uuid=device["uuid"], new_uuid=device["uuid"])
+                error_count = 0
+                for _ in range(10):
+                    data = self.dmg_command.storage_replace_nvme(old_uuid=device["uuid"],
+                                                                 new_uuid=device["uuid"])
+                    time.sleep(5)
+                    if data['error'] or len(data['response']['host_errors']) > 0:
+                        error_count = error_count + 1
```

Comment:
```suggestion
                        error_count += 1
```

Reply:
Minor. Will do if we do any improvements to the script later.